markdownlint.js

```diff
@@ -41,14 +41,22 @@ function readConfiguration(args) {
   return config;
 }
 
-function prepareFileList(files) {
+function prepareFileList(files, fileExtensions) {
   var globOptions = {
     nodir: true
   };
+  var extensionGlobPart = '*';
+  if (fileExtensions && fileExtensions.length > 0) {
+    if (fileExtensions.length === 1) {
+      extensionGlobPart += '.' + fileExtensions[0];
+    } else {
+      extensionGlobPart += '.{' + fileExtensions.join(',') + '}';
```

Comment:
Will this syntax work for just one item? (I’d expect it to, but it needs testing.) If it does, the if/else can be removed.